colin-api/src/colin_api/models/filing.py

```diff
@@ -1402,7 +1401,7 @@ def add_filing(cls, con, filing: Filing, lear_identifier: str) -> int:
             is_frozen_condition = (
                 flags.is_on('enable-bc-ccc-ulc') and
                 business['business']['legalType'] != Business.TypeCodes.COOP.value and
-                cls.is_business_in_lear(lear_identifier)
+                filing_source == cls.FilingSource.LEAR.value
```

Comment:
oh good call, didn't realize the BAR filings were passing "BAR" as source!

Reply:
Thanks Vysakh!